rosdep/python.yaml

```diff
@@ -6003,6 +6010,9 @@ python3-twisted:
   debian: [python3-twisted]
   fedora: [python3-twisted]
   gentoo: [dev-python/twisted]
+  rhel:
+    '*': ['python%{python3_pkgversion}-twisted']
```

Comment:
python3-twisted in EPEL 8 http://rpmfind.net/linux/RPM/epel/8/x86_64/Packages/p/python3-twisted-19.10.0-2.el8.x86_64.html